environments/idvc22.3.0.linux.yml

```diff
@@ -0,0 +1,127 @@
+  - importlib_metadata=4.8.1=hd8ed1ab_1
+  - jpeg=9d=h36c2ea0_0
+  - jsoncpp=1.8.4=hc9558a2_1002
+  - kiwisolver=1.3.1=py36h51d7077_0
+  - krb5=1.17.2=h926e7f8_0
+  - ld_impl_linux-64=2.39=hcc3a1bd_1
+  - libblas=3.9.0=13_linux64_openblas
+  - libcblas=3.9.0=13_linux64_openblas
+  - libclang=9.0.1=default_hde54327_0
+  - libcurl=7.71.1=hcdd3856_8
+  - libedit=3.1.20191231=he28a2e2_2
+  - libev=4.33=h516909a_1
+  - libffi=3.2.1=he1b5a44_1007
+  - libgcc-ng=11.2.0=h1234567_1
+  - libgfortran-ng=7.5.0=h14aa051_20
+  - libgfortran4=7.5.0=h14aa051_20
+  - libglib=2.66.3=hbe7bbb4_0
+  - libgomp=11.2.0=h1234567_1
+  - libiconv=1.16=h516909a_0
+  - liblapack=3.9.0=13_linux64_openblas
+  - libllvm9=9.0.1=he513fc3_1
+  - libnetcdf=4.6.2=hbdf4f91_1001
+  - libnghttp2=1.41.0=h8cfc5f6_2
+  - libogg=1.3.2=h516909a_1002
+  - libopenblas=0.3.18=hf726d26_0
+  - libpng=1.6.37=h21135ba_2
+  - libssh2=1.9.0=hab1572f_5
+  - libstdcxx-ng=12.2.0=h46fd767_19
+  - libtheora=1.1.1=h516909a_1004
+  - libtiff=4.0.10=h9022e91_1002
+  - libuuid=2.32.1=h14c3975_1000
+  - libvorbis=1.3.7=he1b5a44_0
+  - libxcb=1.13=h14c3975_1002
+  - libxkbcommon=0.10.0=he1b5a44_0
+  - libxml2=2.9.10=hee79883_0
+  - libxslt=1.1.33=h31b3aaa_0
+  - libzlib=1.2.11=h36c2ea0_1013
+  - lz4-c=1.8.1.2=0
+  - matplotlib=3.3.4=py36h5fab9bb_0
+  - matplotlib-base=3.3.4=py36h62a2d02_0
+  - natsort=8.2.0=pyhd8ed1ab_0
+  - ncurses=6.2=h58526e2_4
+  - nspr=4.29=he1b5a44_1
+  - nss=3.59=h2c00c37_0
+  - numpy=1.19.4=py36h8732dcd_1
+  - olefile=0.46=pyh9f0ad1d_1
+  - openssl=1.1.1o=h166bdaf_0
+  - packaging=21.3=pyhd8ed1ab_0
+  - pcre=8.44=he1b5a44_0
+  - pillow=6.2.1=py36h6b7be26_0
+  - pip=21.3.1=pyhd8ed1ab_0
+  - pthread-stubs=0.4=h36c2ea0_1001
+  - pyparsing=3.0.9=pyhd8ed1ab_0
+  - pyqt=5.12.3=py36haa643ae_3
+  - pyside2=5.13.2=py36hbee1fde_2
+  - python=3.6.11=h4d41432_2_cpython
+  - python-dateutil=2.8.2=pyhd8ed1ab_0
+  - python_abi=3.6=2_cp36m
+  - pyyaml=5.3.1=py36he6145b8_1
+  - qdarkstyle=3.1=pyhd8ed1ab_0
+  - qt=5.12.5=hd8c4c69_1
+  - qtpy=2.0.1=pyhd8ed1ab_0
+  - readline=8.0=he28a2e2_2
+  - schema=0.7.5=pyhd8ed1ab_0
+  - setuptools=58.0.4=py36h5fab9bb_2
+  - six=1.16.0=pyh6c4a22f_0
+  - sqlite=3.33.0=h4cf870e_1
+  - tbb=2020.2=hc9558a2_0
+  - tk=8.6.11=h21135ba_0
+  - tornado=6.1=py36h1d69622_0
+  - typing_extensions=4.1.1=pyha770c72_0
+  - vtk=8.1.2=py36h8cc7ba1_1200
+  - wheel=0.37.1=pyhd8ed1ab_0
+  - xorg-kbproto=1.0.7=h14c3975_1002
+  - xorg-libice=1.0.10=h516909a_0
+  - xorg-libsm=1.2.3=h84519dc_1000
+  - xorg-libx11=1.6.12=h36c2ea0_0
+  - xorg-libxau=1.0.9=h14c3975_0
+  - xorg-libxdmcp=1.1.3=h516909a_0
+  - xorg-libxt=1.2.0=h516909a_0
+  - xorg-xproto=7.0.31=h14c3975_1007
+  - xz=5.2.5=h516909a_1
+  - yaml=0.2.5=h516909a_0
+  - zipp=3.6.0=pyhd8ed1ab_0
+  - zlib=1.2.11=h36c2ea0_1013
+  - zstd=1.3.3=1
+  - pip:
+    - cil-data==22.0.0
+    - pyqt5-sip==4.19.18
+    - pyqtchart==5.12
+    - pyqtwebengine==5.12.1
+prefix: /home/m1116300/.conda/envs/base-env/envs/idvc
```

Comment:
these are suspicious

Reply:
https://stackoverflow.com/questions/41274007/anaconda-export-environment-file

Seems like it's fine to leave but I will remove and test again